src/Symfony/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -50,6 +50,7 @@ final class PublishMercureUpdatesListener
         'topics' => true,
         'data' => true,
         'private' => true,
+        'private_fields' => true,
```

Comment:
what are `private_fields`? couldn't find that in the specification.

Reply:
These are fields that are also read into the fields array based on witch the sha256 hash for the subscription id is generated. In the REST side of things this is called "Issuing restrictive updates": https://api-platform.com/docs/core/mercure/#dispatching-restrictive-updates-security-mode
There you just generate a topic prefix, but since topics are not part of GraphQL subscription, we are incorporating value of the field  so it generates unique subscription id that works only for that specific combo of private fields.